src/__tests__/integ.global-data-ingestion.test.ts

```diff
@@ -0,0 +1,106 @@
+import { SynthUtils } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
+import * as cf from '@aws-cdk/aws-cloudfront';
+import * as origins from '@aws-cdk/aws-cloudfront-origins';
+import * as iam from '@aws-cdk/aws-iam';
+import * as firehose from '@aws-cdk/aws-kinesisfirehose';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as cdk from '@aws-cdk/core';
+
+import * as extensions from '../extensions';
+
+test('minimal usage', () => {
+  // GIVEN
+  const deliveryStreamName = 'test';
+  const app = new cdk.App();
+  const stack = new cdk.Stack(app, 'global-data-ingestion');
+
+  // WHEN
+  /**
+   * Kinesis Firehose components
+   */
+  const firehoseRole = new iam.Role(stack, 'FirehoseRole', {
+    assumedBy: new iam.ServicePrincipal('firehose.amazonaws.com'),
+  });
+
+  const firehoseDataBucket = new s3.Bucket(stack, 'FirehoseDataBucket', {
+    autoDeleteObjects: true,
+    removalPolicy: cdk.RemovalPolicy.DESTROY,
+  });
+  firehoseDataBucket.grantReadWrite(firehoseRole);
+
+  const firehoseStream = new firehose.CfnDeliveryStream(stack, 'FirehoseStream', {
+    deliveryStreamName,
+    deliveryStreamType: 'DirectPut',
+    extendedS3DestinationConfiguration: {
+      bucketArn: firehoseDataBucket.bucketArn,
+      bufferingHints: {
+        intervalInSeconds: 60,
+        sizeInMBs: 5,
+      },
+      roleArn: firehoseRole.roleArn,
+      errorOutputPrefix: 'error',
+    },
+  });
+
+  /**
+   * Cloudfront and Lambda@Edge
+   */
+  const ext = new extensions.GlobalDataIngestion(stack, 'GlobalDataIngestion', {
+    firehoseStreamName: firehoseStream.deliveryStreamName ?? 'gg',
```

Comment:
```suggestion
    firehoseStreamName: firehoseStream.deliveryStreamName!
```
I believe the `deliveryStreamName` should exist as you already defined it in line 14 and 33.